src/Kernel-Tests/NumberTest.class.st

```diff
@@ -142,8 +142,10 @@ NumberTest >> testRaisedToIntegerWithFloats [
 { #category : #tests }
 NumberTest >> testReadFrom [
 	
-	self assert: 1.0e-14	= (Number readFrom: '1.0e-14').
-	self assert: 2r1e26	= (Number readFrom: '2r1e26').
+	self assert: 1.0e-14    = (Number readFrom: '1.0e-14').
+	"Use a literal number format that compiles in GemStone.
+	 Note that the test still checks the same value."
+	self assert: 16r4000000 = (Number readFrom: '2r1e26').
 	self should: [Number readFrom: 'foo'] raise: Error
```

Comment:
GemStone doesn't allow radix 2 floating point numbers (2r1e26), so I've changed the numeric literal to 16r4000000. The test works the same since we are testing the Number class >> readFrom: method. With this change the test still tests the same thing in Pharo and at least compiles in GemStone.

Reply:
ok